README.md

```diff
@@ -196,6 +196,9 @@ The Datadog implementation support tags, which you can use to slice and dice met
 ``` ruby
 StatsD.increment('my.counter', tags: ['env:production', 'unicorn'])
 GoogleBase.statsd_count :insert, 'GoogleBase.insert', tags: ['env:production']
+
+# Use a lambda for generating dynamic tags from the caller or method arguments
+StatsD.increment('my.counter', tags: lambda { |calle, args| ['dynamic tag'])
```

Comment:
I don't see where this is implemented.

I don't actually mind it not being there, because it's not useful outside of the metaprogramming methods and only makes the implementation more complex and slower. I'd like to keep the basic `StatsD.increment` call as fast as possible, because it is in many hot code paths.